Source/Windows/Common/PortabilityInfo.h

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+#pragma once
+#include "Common/Config.h"
+
+namespace FEX {
+static inline FEX::Config::PortableInformation ReadPortabilityInformation(const fextl::string& ExecutablePath) {
+  const FEX::Config::PortableInformation BadResult {false, {}};
+  const char* PortableConfig = getenv("FEX_PORTABLE");
+  if (!PortableConfig) {
+    return BadResult;
+  }
+
+  uint32_t Value {};
```

Comment:
this would be simpler:
```
const char* PortableConfig = getenv("FEX_PORTABLE");
if (!PortableConfig || strtol(PortableConfig, nullptr, 0) == 0) {
  return {false, {}};
} else {
...
}
```

Reply:
Sure thing, changed over.